src/main/java/org/commonjava/util/partyline/impl/local/RandomAccessJFS.java

```diff
@@ -46,8 +47,8 @@ public LocalLockManager getLocalLockManager()
     }
 
     @Override
-    public void updateDominantLocks( String path, LockLevel level )
+    public void updateDominantLocks( String path, UnlockStatus unlockStatus )
     {
-        // TODO: this is handled by lock owner?
+        assert true;
```

Comment:
I don't think this is going to help us, and it's possible it'll have some sort of overhead cost. Let's just leave the method empty.